Tasks/DownloadGitHubReleaseV0/task.json

```diff
@@ -94,6 +99,15 @@
         "repositoryName": "$(userRepository)",
         "releaseByTagName": "$(defaultVersionType)"
       }
+    },
+    {
+      "target": "userRepositorySearch",
+      "endpointId": "$(connection)",
+      "dataSourceName": "SearchRepositoriesByName",
+      "resultTemplate": "{ \"Value\" : \"{{full_name}}\", \"DisplayValue\" : \"{{full_name}}\" }",
+      "parameters": {
+        "searchText": "$(name)"
```

Comment:
```suggestion
        "searchText": "$(version)"
```

Reply:
What is this?

Reply:
Not sure where it will pick the **name** from, do we expect input in **version** to be passed as **searchText**?

Reply:
Not required as discussed.